src/geop/services/src/main/scala/com/azavea/mmw/GeopServiceActor.scala

```diff
@@ -32,23 +28,57 @@ object Loader {
 
   import scala.util._
 
-  import DataFiles._
   import Histogram._
 
+  val bucket = "com.azavea.datahub"
+  val prefix = "catalog"
+  val layerName = "nlcd-tms"
+  val layerZoom = 13
```

Comment:
I got the bucket, prefix, layer name, and layer zoom from the information that you provided in the card.  My understanding is that 13 is the highest-resolution that we have?  We seem to be encountering issues with the speed of drawing these from S3 -- if that persists we may want to go to a lower resolution?